composer.json

```diff
@@ -108,6 +108,7 @@
     "drupal/module_filter": "^4",
     "drupal/openid_connect": "3.x-dev",
     "drupal/queue_ui": "^3",
+    "drupal/roleassign": "^2.0",
```

Comment:
@hosomitm2nd Please keep the version style with the others:

```suggestion
    "drupal/roleassign": "^2",
```

Reply:
@naoi
I fixed the version style.
Please review again.
Thank you.